packages/protocol/contracts/common/linkedlists/AddressSortedLinkedListWithMedian.sol

```diff
@@ -155,16 +139,15 @@ library AddressSortedLinkedListWithMedian {
    * @notice Gets all elements from the doubly linked list.
    * @return An unpacked list of elements from largest to smallest.
    */
-  function getElements(
-    SortedLinkedListWithMedian.List storage list
-  )
+  function getElements(SortedLinkedListWithMedian.List storage list)
     public
     view
     returns (address[] memory, uint256[] memory, SortedLinkedListWithMedian.MedianRelation[] memory)
   {
     bytes32[] memory byteKeys = list.getKeys();
     address[] memory keys = new address[](byteKeys.length);
     uint256[] memory values = new uint256[](byteKeys.length);
+    // prettier-ignore
```

Comment:
Why do we need this?

Reply:
Prettier was unable to produce the line below, instead trying to put everything on a single line except the function argument which then violated out max-line-length on our lint, so this seemed like the best trade-off